src/Movie-details/MovieDetails.js

```diff
@@ -46,14 +47,27 @@ class MovieDetails extends Component {
       backdrop: movie.backdrop_path,
       releaseDate: this.formatDate(movie.release_date),
       overview: movie.overview,
-      averageRating: this.props.formatAverageRating(movie.average_rating),
+			averageRating: this.props.formatAverageRating(movie.average_rating),
+			userRating: this.findUserRating(this.props),
```

Comment:
Rad.

Reply:
LOL How did this happen!? I think we will need to run the linter here soon to fix some of the weird indentation VSCode does.